package.json

```diff
@@ -4,28 +4,27 @@
   "private": true,
   "license": "MIT",
   "engines": {
-    "node": ">= 16.4.0"
+    "node": "^20.19.0"
```

Comment:
```suggestion
    "node": "^20.19.0"
```

Don't upgrade `engines` here until we branch for v8. Should be fine if our dev environments require Node 20.19 temporarily while end consumers still use v16.4 code.

Reply:
<img width="773" height="144" alt="image" src="https://github.com/user-attachments/assets/43aa38d6-9816-4d49-b06e-27cd99dedc65" />
just to confirm the ask here is to change this back to `>= 16.4.0`?  I'm getting a bit thrown off by the suggested change preview